code/script.js

```diff
@@ -1,13 +1,25 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
+const inputWrapper = document.getElementById('input-wrapper')
+const nameInput = document.getElementById('name-input')
+const button = document.getElementById("send-btn")
+const nameForm = document.getElementById("name-form")
 // Global variables, if you need any, declared here
+let questionNumber = 1
+
 
 // Functions declared here
+const botAnswer = (message) =>{
+  showMessage(message, 'bot')
+}
 
+const userAnswer = (message) => {
+  showMessage(message, 'user')
+}
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log('reply')
```

Comment:
Good with the console.logs, I tried it too after a while just to make sure the functions was being called to the console